### PR TITLE
Issue #386 -Use browsers defined in General/Web Browser for debugging

### PR DESCRIPTION
**Pull request by mickaelistria**
Signed-off-by: Mickael Istria <mistria@redhat.com>

### CODE REVIEW

**Comment by mickaelistria** [comment]
@AObuchow can you please verify this patch works in the case of the Chrome vs Chromium case?

**Comment by AObuchow** [comment]
Yes, I’ll review it today :)

**Comment by AObuchow** [comment]
@mickaelistria I can confirm this works in the case of the Chrome vs Chromium for both attach and launch debug configurations :) I still haven't checked out the actual code though.

File: org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/debug/chrome/ChromeRunDAPDebugDelegate.java [hunk 1]
```diff
@@ -116,28 +114,33 @@ static File findDebugAdapter() {
...
+			res = configuration.getAttribute(RUNTIME_EXECUTABLE, "chromium-browser"); //$NON-NLS-1$
 		} catch (CoreException e) {
 			IStatus errorStatus = new Status(IStatus.ERROR, Activator.PLUGIN_ID, e.getMessage(), e);
 			Activator.getDefault().getLog().log(errorStatus);
 		}
+		if (new File(res).isAbsolute()) {
+			return res;
+		}
+		if (res == null || res.isEmpty()) {
+			res = "chromium-browser"; //$NON-NLS-1$
```
Comment (AObuchow) [inline org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/debug/chrome/ChromeRunDAPDebugDelegate.java:128]: Just to make sure I understand, inthe case that [default] is selected, chromium will be used? (I approve of this choice)

**Comment by mickaelistria** [comment]
Thanks!

**Pull request merged**